1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts

```diff
@@ -0,0 +1,14 @@
+function reverse(str: string) {
+  return str.split("").reverse().join("");
+}
+
+function removeSpaces(str: string) {
+  return str.replace(/\s/g, "");
+}
+
+export const isPalindrome = (str: string): boolean => {
+  const reversedStr = reverse(str);
+  return (
+    removeSpaces(reversedStr.toLowerCase()) === removeSpaces(str.toLowerCase())
```

Comment:
Optionally moving toLowercase a generic function along with remove spaces will simplify the isPalindrome